tools/ioflow/ioflow-read.c

```diff
@@ -0,0 +1,225 @@
+/* 
+ * ioflow-read.c        C source code for read tracer. 
+ *
+ *
+ * Copyright (c) Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ *
+ * 25-Aug-2020   Haoning Chen   Created this.
+ */
+
+#include <uapi/linux/ptrace.h>
+#include <linux/blkdev.h>
+#include <linux/fs.h>
+
+#define IO_FLOW_READ 1
+// Use python string replacement to import common header code
+[IMPORT_COMM]
+
+BPF_HASH(map, u64, struct val_t);    // Map of syscall data, with pid as key
```

Comment:
map & rqmap could use more descriptive names, noticed it was a bit confusing reading it in the functions down below

Reply:
Make sense. I changed the names of the two maps to "syscall_map" and "request_map".